care/emr/migrations/0030_facilityorganization_has_children_old_and_more.py

```
@@ -0,0 +1,64 @@
+# Generated by Django 5.1.4 on 2025-04-29 10:13
+
+from django.db import migrations, models
+
+def recursively_copy_and_fix_has_children_value(row):
+    metadata = row.metadata
+    govt_org_type = metadata["govt_org_type"]
+    if govt_org_type == "ward":
+        if row.has_children == False:
```

Ruff (0.8.2)

Avoid equality comparisons to `False`; use `if not row.has_children:` for false checks

Replace with `not row.has_children`

(E712)